lib/src/test/java/com/basistheory/android/service/BasisTheoryElementsTests.kt

```diff
@@ -545,9 +529,64 @@ class BasisTheoryElementsTests {
         verify { sessionsApi.create() }
     }
 
+    @Test
+    fun `getToken should call java SDK without api key override`() = runBlocking {
+        every { provider.getTokensApi(any()) } returns tokensApi
+        every { tokensApi.getById(any())} returns Token()
+
+        bt.getToken("")
```

Comment:
It would be better to test with some random data for the token id instead of verifying "" gets passed through on this and the next test.